flink-runtime-web/web-dashboard/src/app/app.interceptor.ts

```diff
@@ -32,7 +32,7 @@ export class AppInterceptor implements HttpInterceptor {
      * Error response from below url should be ignored
      */
     const ignoreErrorUrlEndsList = ['checkpoints/config', 'checkpoints'];
-    const ignoreErrorMessage = ['File not found.'];
+    const ignoreErrorMessage = ['File not found.', 'Unable to load requested file /jars.'];
```

Comment:
While this does cover the case of multiple error messages when accessing the page, the user still gets odd error messages if he tries to use the submit page in any way.

If we want to backport the proper solution (See subtasks of FLINK-13799) to 1.9, then I would skip this PR.
If we don't want to backport it, then we need a more extensive fix imo.

Reply:
the upload button in the submit page is already hidden when getting the error message, so the user could not use the submit page in the case.
I think we should wait for the proper solution.